lib/ansible/utils/path.py

```diff
@@ -127,7 +127,7 @@ def cleanup_tmp_file(path, warn=False):
                 elif os.path.isfile(path):
                     os.unlink(path)
             except Exception as e:
```

Comment:
Looks like this should handle an `OSError` separately from suppressing everything. It would bring more clarity into what's that mysterious exception with a sometimes-available `errno`.

Reply:
Or even drop that altogether since the outer try-except will suppress it regardless.